src/load

```diff
@@ -211,6 +215,14 @@ def fetch_data_feed(appctx, data_name, args):
 
     # download the file
     data_url = appctx.config.get_param(f'fetch.url.{data_name}', None)
+
+    if data_name == 'epss':
+        today = datetime.now().astimezone(pytz.timezone('UTC'))
+        previous_day = today - timedelta(days=1)
+        # 1.a. When the specified data to be loaded is cve then first download the EPSS data using the previous day from the current date.
+        # (i.e if the run time is 1st Jan 2024 then the data to be downloaded from is https://epss.cyentia.com/epss_scores-2023-12-31.csv.gz
+        data_url = f"{data_url}epss_scores-{previous_day.strftime('%Y-%m-%d')}.csv.gz"
```

Comment:
the separation between {data_url} and `epss_scores` should be done here with `/` and should not rely on the settings from the config.